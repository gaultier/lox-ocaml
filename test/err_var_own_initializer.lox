var a = a; // expect: Cannot read variable `a` in its own initializer
