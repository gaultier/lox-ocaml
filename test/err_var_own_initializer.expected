Cannot read variable `a` in its own initializer
